test/rules/cascade_invocations.dart

```diff
@@ -17,15 +24,27 @@ int get someThing => 5;
 void noCascadeIntermediate() {
   List<int> list = [];
   print(list);
-  list.removeWhere((i) => i % 5 == 0);
+  list.clear();
   list.clear(); // LINT
+  list.toString(); // LINT
+  // toString is not void and it is not a problem.
+}
+
+void cascadeIntermediate() {
+  List<int> list = [];
+  print(list);
+  list.clear();
+  list.clear(); // LINT
+  list..clear(); // LINT
+  list..clear(); // OK
```

Comment:
:O I did not know cascades well at that time, you are totally right we must lint those cases also!